aiida/orm/nodes/process/calculation/calcjob.py

```diff
@@ -373,7 +373,6 @@ def get_retrieve_singlefile_list(self):
             Will be removed in `v2.0.0`, use
             :meth:`aiida.orm.nodes.process.calculation.calcjob.CalcJobNode.get_retrieve_temporary_list` instead.
         """
-        warnings.warn('method is deprecated, use `get_retrieve_temporary_list` instead', AiidaDeprecationWarning)  # pylint: disable=no-member
```

Comment:
Could we move this warning into the other file then, possibly? - In the new (and very much needed) if block.

Reply:
The thing is, this function is most likely never called by the user and if they do, they will have called the `set_retrieve_singlefile_list`. But even that function they never really call, because that is done automatically by the engine. The problem is that the `set_retrieve_singlefile_list` call I can make conditional in the engine only when the user specifies it. But the `get_retrieve_singlefile_list` always has to be called by the engine, so this warning would always show up

Reply:
So let's hit what we can, by the philosophy that if a user manually sets the `retrieve_singlefile_list` a warning occurs, elsewhere, not?

Reply:
At least then, if a user forces the use of this, they are notified. Or did I misunderstand and it will be flashed every time the engine calls it as well?

Reply:
that's the thing, the warning does not make a distinction between who calls it. But in any case, the engine is the only one who is supposed to call it anyway. The user just specified it in `CalcInfo.retrieve_singlefile_list` which they return from the `CalcJob.prepare_for_submission`.

Reply:
Hmm. Okay. Then either we create a proxy-function that includes the warning, which the engine can go around, or we remove the message as you have done, I guess? And I don't like the first option at all, honestly.